src/gui/MainWindow.cpp

```diff
@@ -738,6 +741,7 @@ void MainWindow::saveWidgetState( QWidget * _w, QDomElement & _de )
 	_de.setAttribute( "visible", visible );
 	_de.setAttribute( "minimized", _w->isMinimized() );
 	_de.setAttribute( "maximized", _w->isMaximized() );
+	_de.setAttribute( "fullscreen", _w->isFullScreen() );
```

Comment:
This should also be removed, according to logic in previous comments.

Reply:
Could you explain why? I'm not 100% sure what it does but I saw maximise & minimise there so I kept this as well. Does it save the "fullscreen" argument in the project file by chance?

Reply:
Essentially yes, I believe that's what it does. QDomElement is "[an] element in the DOM tree", and "Document Object Model (DOM) [...] treats an XML or HTML document as a tree structure". In other words, anything involving a QDomElement has to do with de/serialization, as far as I'm aware.

Since we have an MDI interface, individual windows inside the main LMMS window can have minimize/maximize states, which is what's saved here. Fullscreen is a property of the top level LMMS window though, so even if it were saved it shouldn't be once per widget.

Hope this isn't too terrible of an explanation, haha.

Reply:
Ohh, I see. Basically I was saving the fullscreen attribute for widgets which didn't even have the fullscreen option xD

Thanks for the explanation, this will probably help in future PRs.